wallet/src/test/java/io/neow3j/wallet/Nep5Test.java

```diff
@@ -0,0 +1,72 @@
+package io.neow3j.wallet;
+
+import io.neow3j.contract.ScriptHash;
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.http.HttpService;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.math.BigInteger;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.fail;
+
+public class Nep5Test {
```

Comment:
@Priyanshu-soni in the report of your last commit (https://github.com/neow3j/neow3j/pull/99#issuecomment-556003123), the test coverage decreased by 0.3%. We always should keep the same coverage level or increase with pull requests. Can you check which part of the code that is not covered, somehow?